### DISCUSSION TITLE
A suggestion to archiving the protests

### DISCUSSION THREAD

**Discussion by sepehrnava** [WeAreMahsaAmini/Core#175] [Ideas]
Hello, I am so happy that you have created this repo for discussing helping our people from abroad.
Last week I was wondering what to do to help the process, one day I decided to develop this website:

https://eteraznegar.vercel.app/

You can view the archive of all the protests that happened in Iran by provinces and cities with a date filter.
For instance, you can click on Mazandaran province to see the data.

this idea is in its early stages and is not functioning. If you think it would help inform people, let's discuss it, we can develop it and make it English.

**Comment by danig87** [comment]
This is really cool! I think it'd be nice to be able to view all videos in a date range, or select multiple or all cities and watch all the videos from today for example.
